Packs/Malware/README.md

```diff
@@ -1,21 +1,21 @@
-Malware, or malicious software, is any program or file that is intentionally designed to be harmful to a computer, server, client or computer network.
+Malware, or malicious software, is any program or file that is intentionally designed to be harmful to a computer, server, client, or computer network.
 Malware investigations require security teams to reconcile data from multiple malware analysis and threat intelligence tools. Valuable time is lost shuttling between screens and executing repeatable tasks while an attack continues to manifest.
-This Malware content pack contains the ‘Endpoint Malware Investigation - Generic’ playbook, that automates response to a malware found on an endpoint. The pack also contains the corresponding custom malware incident fields, views and layouts to facilitate analyst investigation.  
+This Malware content pack contains the ‘Endpoint Malware Investigation - Generic’ playbook, that automates response to a malware found on an endpoint. The pack also contains the corresponding custom malware incident fields, views, and layouts to facilitate analyst investigation.  
 The Malware playbooks orchestrate across multiple products to extract and enrich IOCs, detonate malicious files, hunt for more IOCs within the organization, and perform remediation on the malware.
```

Comment:
@evisochek remediate the malware? maybe you meant the host

Reply:
Fixed